### PR TITLE
[fix](fragment mgr) fix the bug that g_fragmentmgr_prepare_latency update uncorrectly

### PR DESCRIPTION
**Pull request by levy5307**
# Proposed changes

Issue Number: close #xxx

## Problem summary

In the scope of `{}`, the value of `duration_ns` is 0. So this will make `g_fragmentmgr_prepare_latency` update uncorrectly.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
I wanted to add a unit test to this function, but found it difficult to implement.
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [x] Is this PR support rollback (If NO, please explain WHY)

### CODE REVIEW

**Comment by liutang123** [comment]
LGTM

**Comment by BiteTheDDDDt** [comment]
run buildall

File: be/src/runtime/fragment_mgr.cpp [hunk 1]
```diff
@@ -753,12 +753,12 @@ Status FragmentMgr::exec_plan_fragment(const TExecPlanFragmentParams& params,
         {
             SCOPED_RAW_TIMER(&duration_ns);
             auto prepare_st = context->prepare(params);
-            g_fragmentmgr_prepare_latency << (duration_ns / 1000);
             if (!prepare_st.ok()) {
                 context->close_if_prepare_failed();
                 return prepare_st;
             }
         }
+        g_fragmentmgr_prepare_latency << (duration_ns / 1000);
```
Comment (wangbo) [inline be/src/runtime/fragment_mgr.cpp:761]: what does ```<<``` mean here? same as ```=```?

Comment (levy5307) [reply be/src/runtime/fragment_mgr.cpp:761]: No, it's append-like operation. You can have a look at the func `operator<<` in `https://github.com/apache/brpc/blob/master/src/bvar/latency_recorder.cpp`

**Review by wangbo — APPROVED** [review]
LGTM

**Comment by levy5307** [comment]
run buildall

**Comment by levy5307** [comment]
run p0

**Comment by levy5307** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**